ProcessLib/ComponentTransport/ComponentTransportFEM.h

```diff
@@ -33,13 +33,22 @@ template <typename NodalRowVectorType, typename GlobalDimNodalMatrixType>
 struct IntegrationPointData final
 {
     IntegrationPointData(NodalRowVectorType const& N_,
-                         GlobalDimNodalMatrixType const& dNdx_,
+            GlobalDimNodalMatrixType const& dNdx_,
```

Comment:
Is it planed to use different shape matrices for the p process and the C process? If it is always the same we could save a lot of memory when the implementation would use only one set of shape matrices.